docs/technical-docs/zotero.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: default
+title: Zotero
+parent: Technical Documentation
+nav_order: 5
+---
+
+# Zotero Tutorial
+
+An overview of what Zotero does, how to get started, and how to add to it.
+
+
+
+#### TABLE OF CONTENTS
+1. [What is Zotero?](https://ndclab.github.io/wiki/docs/zotero.html#what-is-zotero?)
+2. [Set-Up](https://ndclab.github.io/wiki/docs/zotero.html#set-up)
+3. [Saving Papers](https://ndclab.github.io/wiki/docs/zotero.html#saving-papers)
+4. [Folder Structure](https://ndclab.github.io/wiki/docs/zotero.html#folder-structure)
+5. [Creating and Navigating Tags](https://ndclab.github.io/wiki/docs/zotero.html#creating-and-navigating-tags)
+6. [Quick Tips](https://ndclab.github.io/wiki/docs/zotero.html#quick-tips)
+
+
+
+## What is Zotero?
+
+Zotero is a reference manager that allows the lab to have a shared collection of papers and makes it easy for users to add properly formatted in-text citations and bibliographies. 
+
+
+
+## Set-Up
+
+### Create an account
+
+Create an account at this link: https://www.zotero.org/user/register 
+
+### Get added to the NDC_Lab group library
+
+Send a message to the lab manager with either your Zotero username or the email you used to create an account. They will send you an email invite via Zotero to join the group library.
+
+### Install the application and Chrome extension
+
+While you wait to be added to the group library, download both the Zotero application and Chrome extension at this link: https://www.zotero.org/download/ 
+
+Once the application is installed, it should automatically install a plugin for Word, at which point you will need to restart Word. If it doesn't auto-install, see this documentation: https://www.zotero.org/support/word_processor_plugin_manual_installation
+
+### Sync your account
+
+To sync your account to the application, click on Zotero > Preferences > Sync. Click "Link account" and log in using your Zotero username and password. Zotero should automatically update the group library as others add papers to it. However, Zotero can be manually synced via the circular green arrow at the top right of the application window. 
```

Comment:
This has to be done on the app itself, right?  (Not the web browser?)  You might want to make that explicit.

Reply:
Is changing it to "To sync your account to the application, click on Zotero > Preferences > Sync in the application" more clear? I feel like the first part of the first sentence was establishing it to be in the app, but if you were confused, then others will be too! Let me know if you have other suggestions.

Reply:
Yeah, that sounds good!  It wasn't that I was confused myself, TBH, but I anticipate some people staying inside the web interface and getting potentially getting confused that they need to switch over the app if we aren't explicit about it. ;)